flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TableFactoryHarness.java

```diff
@@ -95,7 +95,11 @@
  *    }
  * }
  * }</pre>
+ *
+ * @deprecated If you're using this test utility, then move the test class in flink-table-tests and
+ *     use the TableFactoryHarness from table-test-utils
  */
+@Deprecated
 public class TableFactoryHarness {
```

Comment:
We still have duplicate classes. We should port the existing tests and remove this class.